commons/src/main/java/com/powsybl/commons/datasource/DataSource.java

```diff
@@ -18,4 +23,16 @@ public interface DataSource extends ReadOnlyDataSource {
 
     OutputStream newOutputStream(String suffix, String ext, boolean append) throws IOException;
 
+    static DataSource fromPath(Path file) {
```

Comment:
Maybe it could be useful to have a `fromPath(String file)` which does the `Path.of`?

Reply:
I could but I won't be able to test it

Reply:
With that, I was trying to make it simpler to write than
```
Network network = Network.read(new MultipleReadOnlyDataSource(
        DataSource.fromPath(Path.of("20171002T0930Z_BE_EQ_2.zip")),
        DataSource.fromPath(Path.of("20171002T0930Z_1D_BE_SSH_2.zip")),
        DataSource.fromPath(Path.of("20171002T0930Z_ENTSO-E_EQ_BD_2.zip")),
        DataSource.fromPath(Path.of("20171002T0930Z_ENTSO-E_TP_BD_2.zip")),
        DataSource.fromPath(Path.of("20171002T0930Z_1D_BE_TP_2.zip"))));
```

We could go further and add a constructor to MultipleReadOnlyDataSource with several Paths?

Reply:
That would be nice but would require to change "read" to something else because method would have same erasure that read(List<DataSource>)

Reply:
To get something like:
```java
Network network = Network.read(Path.of("20171002T0930Z_BE_EQ_2.zip"),
        Path.of("20171002T0930Z_1D_BE_SSH_2.zip"),
        Path.of("20171002T0930Z_ENTSO-E_EQ_BD_2.zip"),
        Path.of("20171002T0930Z_ENTSO-E_TP_BD_2.zip"),
        Path.of("20171002T0930Z_1D_BE_TP_2.zip"));
```

Reply:
Path array overload added